detection_rules/schemas/v7_9.py

```diff
@@ -50,7 +51,7 @@ class ApiSchema79(ApiSchema78):
     author = jsl.ArrayField(jsl.StringField(default="Elastic"), required=True, min_items=1)
     building_block_type = jsl.StringField(required=False)
     exceptions_list = jsl.ArrayField(required=False)
-    license = jsl.StringField(required=True, default="Elastic License")
+    license = jsl.StringField(required=True, default="Elastic License v2")
```

Comment:
makes sense to bump this in the 7.8 schema, when the field was introduced, in case a rule is created and validated with an older schema